source/fundamentals/logging.txt

```diff
@@ -0,0 +1,151 @@
+``LoggingSettings`` class, then assign it to the ``LoggingSettings`` property of your 
+``MongoClientSettings`` object.
+
+The ``LoggingSettings`` constructor accepts the following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Property
+     - Description
+
+   * - ``LoggerFactory``
+     - | The ``ILoggerFactory`` object that creates an ``ILogger``. You can create 
+         an ``ILoggerFactory`` object by using the ``LoggerFactory.Create()`` method.
+       |
+       | **Data Type**: `ILoggerFactory <https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.iloggerfactory?view=dotnet-plat-ext-7.0>`__
+       | **Default**: ``null``
+
+   * - ``MaxDocumentSize``
+     - | Optional. The maximum number of characters for extended JSON documents in logged 
+         messages.
+       |
+       | For example, when the driver logs the ``CommandStarted`` message, it truncates 
+         the ``Command`` field to the number of characters specified in this parameter.
+       |
+       | **Data Type**: {+int-data-type+}
+       | **Default**: ``1000``
+
+The following code sample shows how to create a ``MongoClient`` that 
+logs all debug messages to the console: 
+
+.. code-block:: csharp 
+    
+   using var loggerFactory = LoggerFactory.Create(b =>
+   {
+     b.AddSimpleConsole();
+     b.SetMinimumLevel(LogLevel.Debug);
+   });
+ 
+   var settings = MongoClientSettings.FromConnectionString("<your connection string>");
+   settings.LoggingSettings = new LoggingSettings(loggerFactory);
+   var client = new MongoClient(settings);
+
+Log Messages by Category
+------------------------
+
+Each message generated by a MongoDB cluster is assigned a *category*. This lets you 
+specify different log levels for different types of messages. 
+
+MongoDB uses the following categories to classify messages:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Category
+     - Description
+
+   * - ``MongoDB.Command``
+     - | The progress of commands run against your cluster, represented by the 
+         ``CommandStarted``, ``CommandSucceeded``, and ``CommandFailed`` events
+
+   * - ``MongoDB.SDAM``
+     - | Changes in the topology of the cluster, including the 
+         ``ClusterAddedServer``, ``ClusterRemovedServer``, ``ServerHeartbeatStarted``,
+          ``TopologyDescriptionChanged``, and ``ServerDescriptionChangedEvent`` events 
+
+   * - ``MongoDB.ServerSelection``
+     - | The decisions that determine which server to send a particular command to
+
+   * - ``MongoDB.Connection``
+     - | Changes in the cluster connection pool, including the ``PoolReady``, 
+         ``PoolClosed``, ``ConnectionCreated``, and ``ConnectionCheckout`` events 
+
+   * - ``MongoDB.Internal.*``
+     - | Prefix for all other {+driver-short+} internal components
+
+.. tip::
+ 
+   To log messages from all components, use the ``Default`` category. 
```

Comment:
The `Default` category, is just used for specify the default minimal logs verbosity in configuration.
Also each logging provider (Console, File,...), might have different default verbosity filter.
For example by default, without any configuration, `SimpleConsole` provider will display all messages with verbosity information and higher, regardless of their category.

So I think we this tip can be removed, or change to: 'Use `Default` category for adjusting the minimal verbosity for all categories in your application"